archive-cleaner

```diff
@@ -122,6 +134,24 @@ def main(repo=DEFAULT_SOURCE_DIR, archive=DEFAULT_ARCHIVE_DIR, keep_years=DEFAUL
                 unlink(f'{path}{SIG_EXT}')
             except FileNotFoundError:
                 print(f'ERROR: file not found for deletion: {path}{SIG_EXT}')
+        else:
+            pkg = basename(path)
+            if pkg in removed:
+                continue
+
+            try:
+                cleanup_size += getsize(path)
+            except FileNotFoundError:
+                print(f'ERROR: file not found for calculation size: {path}')
```

Comment:
If the file can't be found, you should probably not add the file to removed just yet. Instead I'd say it should be added to the set only if the package was removed. I don't mind the signature as much because that is rather small, but the package itself seems important enough to improve calculation. So essentially, I'd suggest that you just put the `removed.add(pkg)` from below into this try block as the last line.